src/SimpleCalculator/UserInputs.cs

```diff
@@ -1,17 +1,23 @@
 namespace SimpleCalculator;
 
 public class UserInputs
-
 {
-    public static int GetUserInput(string message)
+    private string userInput;
```

Comment:
please use a special [naming convention](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case) for private fields